Hardcore.lua

```diff
@@ -1456,9 +1585,14 @@ function Hardcore:GenerateVerificationString()
 		ATTRIBUTE_SEPARATOR)
 	local bubbleHearthIncidentsVerificationString = Hardcore_tableToUnicode(Hardcore_Character.bubble_hearth_incidents)
 	local playedtimeGapsVerificationString = Hardcore_tableToUnicode(Hardcore_Character.played_time_gap_warnings)
-
+	local converted_successfully = "FALSE"
+	if (Hardcore_Character.converted_successfully) then
+		converted_successfully = "TRUE"
+	end
+	local dkInfo = {Hardcore_Character.sacrificed_at, converted_successfully, Hardcore_Character.converted_time}
+	local deathknightVerificationString = Hardcore_join(Hardcore_map(dkInfo,Hardcore_stringOrNumberToUnicode),ATTRIBUTE_SEPARATOR)
 	return Hardcore_join({baseVerificationString, bubbleHearthIncidentsVerificationString,
-							playedtimeGapsVerificationString}, ATTRIBUTE_SEPARATOR)
+							playedtimeGapsVerificationString, deathknightVerificationString}, ATTRIBUTE_SEPARATOR)
```

Comment:
I thought we wanted to make the DK data a table?

```
local converted_successfully = "FALSE"
	if (Hardcore_Character.converted_successfully) then
		converted_successfully = "TRUE"
	end

dk_conversion = {
	sacrificed_at = Hardcore_Character.sacrificated_at,
	converted_successfully = converted_successfully,
	converted_time = Hardcore_Character.converted_time,
}

local deathknightVerificationString = Hardcore_tableToUnicode(dk_conversion)

return return Hardcore_join({baseVerificationString, bubbleHearthIncidentsVerificationString,
							playedtimeGapsVerificationString, deathknightVerificationString}, ATTRIBUTE_SEPARATOR)
```

This way all the data is grouped together and can be converted to json like this:

```
{
"deathknight": {
  	"sacrificed_at": "2022-08-16", // not the exact date format, just for demonstration
	"converted_successfully: true,
	"converted_time: "2022-08-16",
  }
}
```

Reply:
Yeah I wanted to make table, got confused a little bit haven't done lua coding in some time.
I did rework code as I needed to make row with all the necessary data of DK and then insert into table. 
Table to unicode string didn't want to take one record and did output empty string.
Did some debugging to find out that.